### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lipmaneth**
Suggestions for docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,37 @@
...
-Learn more about StoryCheck and how you can integrate it into your testing workflow here.
+Learn more about StoryCheck and how you can integrate it into your testing workflow [here](https://github.com/GuardianUI/storycheck).
 
 ###
 
-### Mission
+## Mission
 
-Our mission is simple. Reduce the risk to users and teams that choose to partake in the crypto space. Improving testing and monitoring tooling is the primary focus at the moment.
+Our mission is simple. We aim to reduce the risk to users and teams that choose to partake in the crypto space. Improving testing and monitoring tooling is the primary focus at the moment.
```
Comment (0xLienid) [inline README.md]: "We aim to reduce the risk to users and teams that choose to partake in the crypto space face." would be better

File: api/gui.md [hunk 2]
```diff
@@ -63,9 +63,9 @@ Sets the allowance of an address to spend the test wallet's ERC20 tokens to a sp
 
 #### Inputs
 
-* `token` - The address of the token to set the allowance on
-* `spenderAddress` - The address to approve to spend the test wallet's tokens
-* `amount` - The amount to approve the spender to spend. This needs to have the appropriate number of decimals (i.e. 1 ETH is "1000000000000000000")
+* `token` - The designated token address for balance allocation (e.g. USDC on Ethereum is [0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48](https://etherscan.io/token/0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48))
+* `spenderAddress` - The address to authorize for spending the tokens in the test wallet.
+* `amount` - The amount to be approve to be spent by the spender. This needs to have the appropriate number of decimals (i.e. 1 ETH is "1000000000000000000")
```
Comment (0xLienid) [inline api/gui.md]: approved

File: end-to-end-testing/writing-your-first-e2e-test.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Writing Your First E2E Test
 
-There are eight phases to writing your first GuardianUI end-to-end test:
+There are eight phases to writing your first e2e test using GuardianTest:
```
Comment (0xLienid) [inline end-to-end-testing/writing-your-first-e2e-test.md]: capitalize E2E

**Review by 0xLienid — CHANGES_REQUESTED** [review]
couple small bits of feedback

**Comment by lipmaneth** [comment]
@0xLienid believe I've addressed your comments.

**Comment by lipmaneth** [comment]
Changes were done in gitbook. This is now old.

**Pull request closed without merging**